src/Log/EventService.php

```diff
@@ -184,6 +189,7 @@ public function getIdentifier(object $entity)
         $identifier = $this->em->getClassMetadata($className)->getSingleIdentifierFieldName();
         $refl = $this->refl->getAccessibleProperty($className, $identifier);
 
+        assert($refl !== null);
```

Comment:
What's the reasoning behind this assert?

Reply:
that if `$refl` is null, you can't possibly hope to call a method on it. yet `getAccessiblePropertert` returns a `ReflectionProperty`.

I suppose you're right if you mean `assert($refl instanceof \ReflectionProperty)` would be nicer. I've updated it acordingly.